components/VM_Arm/src/main.c

```diff
@@ -134,6 +133,9 @@ seL4_CPtr camkes_get_smmu_cb_cap();
 seL4_CPtr camkes_get_smmu_sid_cap();
 #endif
 
+char *get_allocator_mempool(void);
```

Comment:
Could this prototypes move to a template header, so this is re-usable for other implementations also?

Reply:
Sure. Just tested and I can add `TEMPLATE_HEADERS` to add a template header to `camkes.h`.